Cargo.toml

```diff
@@ -210,8 +213,6 @@ members = [
 #
 # This list is ordered alphabetically.
 [profile.dev.package]
-aes-soft = { opt-level = 3 }
-aesni = { opt-level = 3 }
```

Comment:
It looks like we are removing the use of the instruction set, which means slowdown in performance. Is this deliberate or was this part of upstream changes?

Reply:
They are **not** used in this version. Hopefully latest versions of compiler warn about that:
```
❯ cargo b --release
warning: profile package spec `aes-soft` in profile `dev` did not match any packages
warning: profile package spec `aesni` in profile `dev` did not match any packages

        Did you mean `aes`?
   Compiling proc-macro2 v1.0.32
   Compiling syn v1.0.81
   Compiling libc v0.2.107
   Compiling memchr v2.4.1
   Compiling typenum v1.14.0
   Compiling futures-core v0.3.17
   Compiling futures-channel v0.3.17
   Compiling anyhow v1.0.45
   Compiling getrandom v0.1.16
```